.openpublishing.redirection.json

```diff
@@ -4252,6 +4272,26 @@
       "source_path": "docs/visual-basic/programming-guide/concepts/linq/how-to-build-linq-to-xml-examples.md",
       "redirect_url": "/dotnet/standard/linq/build-linq-xml-examples"
     },
+    {
+      "source_path": "docs/visual-basic/programming-guide/concepts/linq/how-to-catch-parsing-errors.md",
+      "redirect_url": "/dotnet/standard/linq/catch-parsing-errors",
+      "redirect_document_id": true
```

Comment:
the ones under vb should be false or removed

Reply:
Ah, I get it, I think. Let me ask this, not as a proposal, but as a point of understanding: since I always use the document id from the c# article in the merged article, do we really need to use redirect_document_id at all?

Reply:
My understanding is that you need that from the BI data to flow into the new topic. There are many GUIDs involved and if you don't add this, a new internal one is created.

Reply:
Otherwise, the new topic is not seen as the old one, but as a brand new one.